src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/CCDCaseRepository.java

```diff
@@ -154,12 +154,13 @@ public Claim saveClaim(String authorisation, Claim claim) {
     }
 
     @Override
-    public void saveClaimDocuments(
+    public Claim saveClaimDocuments(
         String authorisation,
         Long claimId,
         ClaimDocumentCollection claimDocumentCollection
     ) {
         coreCaseDataService.saveClaimDocuments(authorisation, claimId, claimDocumentCollection);
+        return null; //TODO: This method should return a claim.
```

Comment:
when will this be done? will it break anything?

Reply:
can this be actioned pls?

Reply:
It will not break anything for now but when we integrate document upload with CCD this definitely has to be done or it will break. This is being done in John's PR as CCD is out of scope for this PR.

Reply:
I have now changed this to return a claim but this code will probably be changed as part of John's PR as well.